irods_consortium_continuous_integration_build_hook.py

```diff
@@ -6,17 +6,10 @@
 
 import irods_python_ci_utilities
 
-def add_cmake_to_front_of_path():
-    cmake_path = '/opt/irods-externals/cmake3.21.4-0/bin'
-    os.environ['PATH'] = os.pathsep.join([cmake_path, os.environ['PATH']])
-
 def install_building_dependencies(externals_directory):
     externals_list = [
         'irods-externals-boost1.81.0-2',
-        'irods-externals-clang16.0.6-0',
-        'irods-externals-cmake3.21.4-0',
-        'irods-externals-fmt8.1.1-2',
-        'irods-externals-json3.10.4-0'
```

Comment:
These are removed because the server declares dependencies on the distro-provided packages? Basically, we assume they are provided by the system via the server package?

Do/Should we have this plugin declare dependencies on boost, fmt, and json?

Reply:
The library, server, and dev packages already declare the proper dependencies. They are not needed here.

Reply:
Makes sense. Is that the normal/expected way of handling such cases?

Reply:
Got it. Resolving.